exchanges/bybit/convert_test.go

```diff
@@ -0,0 +1,130 @@
+package bybit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+)
+
+func TestGetConvertCoinList(t *testing.T) {
+	t.Parallel()
+
+	_, err := e.GetConvertCoinList(t.Context(), "", currency.EMPTYCODE, true)
+	assert.ErrorIs(t, err, errUnsupportedAccountType)
+
+	_, err = e.GetConvertCoinList(t.Context(), Uta, currency.EMPTYCODE, true)
+	assert.ErrorIs(t, err, currency.ErrCurrencyCodeEmpty)
+
+	if mockTests {
+		t.Skip(skipAuthenticatedFunctionsForMockTesting)
```

Comment:
🤷‍♂️ For me, like ... at least 75% of the advantage of Mocking is that we can offer authenticated testing...

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/1993/commits/c00f4bc8e907e3a449a4b0253af3715f72af6c0a

Mocked what I could didn't have funds to do the actual quote testing atm.